cloudevents-sdk-actix-web/Cargo.toml

```diff
@@ -13,11 +13,11 @@ categories = ["web-programming", "encoding", "web-programming::http-server"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-cloudevents-sdk = { version = "0.3.0", path = ".." }
+cloudevents-sdk = { path = ".." }
```

Comment:
You need to keep the version, it's required when publishing

Reply:
Does this matter if we're going to delete the workspaces? Or do you want to continue to publish all of them even though the logic is replicated in the root? The `cargo publish --dry-run` from the root worked fine.

Reply:
> Or do you want to continue to publish all of them even though the logic is replicated in the root?

Just for another version, then we'll remove them

> https://github.com/vert-x3/vertx-eventbus-bridge-clients/tree/master/Go

you need to try `cargo publish --dry-run` from the subdirs as well

Reply:
Done. I didn't touch `warp` though, since it already lacked a `version`. Never published, I guess?